Job deletion fails when max is higher than total number of jobs

__Steps to reproduce__
1. Set up a user that only has a few jobs associated with it.
2. Call the `jobs delete` command with a `--max` value greater than the number of jobs that exist for that user, like so: `cectl jobs delete --curr 1 --all --max 200`

__Actual result__
An error is produced:
```bash
panic: runtime error: index out of range

goroutine 84 [running]:
github.com/ghchinoy/cectl/cmd.deleteAllJobs.func1(0xc000026b10, 0x2e, 0xc00007a300, 0x60, 0xc000230ae0, 0xc00053cde0, 0xc)
	/home/ghchinoy/go/src/github.com/ghchinoy/cectl/cmd/jobs.go:140 +0x108
created by github.com/ghchinoy/cectl/cmd.deleteAllJobs
	/home/ghchinoy/go/src/github.com/ghchinoy/cectl/cmd/jobs.go:139 +0x283
```

__Expected result__
If there are fewer jobs than provided in the `--max` parameter, simply delete all of the jobs.

--- a/cmd/jobs.go
+++ b/cmd/jobs.go
@@ -135,6 +135,11 @@ func deleteAllJobs(base, auth string) error {
 	} else {
 		max = len(jobs)
 	}
+	// guard against user provided max > len(jobs)
+	if len(jobs) < max {
+		log.Printf("Overriding user requested maximum value of %v to actual maximum %v", max, len(jobs))
+		max = len(jobs)
+	}
 	for j := 0; j < max; j++ {
 		go func(j int) {
 			err := deleteJob(base, auth, jobs[j].ID)

